root/programs/C#/Samples/WebApp_sample/ProjectX_sample/Scripts/touryo/common.js

```diff
@@ -433,50 +490,179 @@ function Fx_DisplayProgressDialog() {
 }
 
 //**********************************************************************************
+// Pseudo Dialog（擬似ダイアログ）
+//**********************************************************************************
+// Ajax：Pseudo Dialog（div）
+var AjaxOKPseudoDialog;
+var AjaxYesNoPseudoDialog;
+
+// Ajax：Pseudo Dialogのサイズ（div）
+var AjaxOKPseudoDialog_Width = 500;
+var AjaxOKPseudoDialog_Height = 300;
+var AjaxYesNoPseudoDialog_Width = 500;
+var AjaxYesNoPseudoDialog_Height = 300;
 
 // ---------------------------------------------------------------
-//  フレームワーク機能（Dialog）
+// OK Pseudo Dialogの初期化
+// ---------------------------------------------------------------
+// 引数    －
+// 戻り値  －
+// ★★★  Dialogのデザインを変える場合は、ここを直接編集
 // ---------------------------------------------------------------
+function Fx_InitPseudoDialog() {
+
+    // divを生成   
+    var _div = null;
+
+    // ---------------------------------------------------------------------
+    // OK Pseudo Dialog ----------------------------------------------------
+     
+    _div = document.createElement("div");
+    _div.id = "AjaxOKPseudoDialog";
+
+    // 幅を指定
+    _div.style.width = AjaxOKPseudoDialog_Width + "px";
+    _div.style.height = AjaxOKPseudoDialog_Height + "px";
+
+    // スタイルを指定
```

Comment:
@yusukemaegawa 

I see, I add it below file.
- \root\programs\C#\Samples\WebApp_sample\ProjectX_sample\Content\touryo\Style.css"